public/app/features/templating/specs/template_srv.test.ts

```diff
@@ -1,609 +0,0 @@
-import { TemplateSrv } from '../template_srv';
```

Comment:
this file was accidentally removed? contains super important tests for templating interpolation :)